data/dashboards/README.md

```diff
@@ -0,0 +1,19 @@
+# README
+
+This folder contains a pre-built summary dashboard for all benchmarks in PerfKitBenchmarker
+(https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/blob/master/README.md)
+.
+
+
+## Setup dashboards
+* Once PerfKitExplorer is deployed, create a BigQuery table containing default machine types for various cloud providers:
+
+        bq load --source_format=NEWLINE_DELIMITED_JSON --replace=true samples_mart.providers providers.json provider_schema.json
+
+* Configure the the dashboards with your BIGQUERY_PROJECT_ID used by PerfKitExplorer by running the following command in this directory:
+
+        sed -i  's/{{ project_id }}/BIGQUERY_PROJECT_ID/g' *.json
```

Comment:
The dashboard included in this PR seems to rely on the default project from the Explorer config.  I'd recommend setting the project_id attribute of the dashboard to BIGQUERY_PROJECT_ID so it gets picked up by this statement.  Otherwise, remove this statement, and document that the user can set the default project for the dashboard on the Dashboard tab of Explorer after opening it.